context/app/static/js/components/detailPage/BulkDataTransfer/Links.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import Divider from '@material-ui/core/Divider';
+
+import { useFlaskDataContext } from 'js/components/Contexts';
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { InfoIcon } from 'js/shared-styles/icons';
+import { LinkContainer, StyledLink } from './style';
+
+function GlobusAccess() {
+  const {
+    entity: { dbgap_study_url, dbgap_sra_experiment_url },
+  } = useFlaskDataContext();
+
+  const linksContentArray = [
+    {
+      title: 'dbGaP Study',
+      url: dbgap_study_url,
+      toolTip: '',
```

Comment:
```suggestion
      tooltip: '',
```
Tooltip as a single word here and elsewhere.